lib/factories/index.js

```diff
@@ -143,25 +143,27 @@ class PowerPointFactory {
         return slide;
     }
 
-    addImage(slide, path, options = {}) {
-        let newImageContentBlock = this.pptFactory.addImage(slide, path, options);
-        let image = new Image({ parentContainer: slide, powerPointFactory: this, content: newImageContentBlock });
+    addImage(slide, source, sourceType, options = {}) {
+        let newImageContentBlock = this.pptFactory.addImage(slide, source, sourceType, options);
+        return new Image({ parentContainer: slide, powerPointFactory: this, content: newImageContentBlock });
```

Comment:
Does this use the native `Image` function or are you overloading it as a custom function?